mlir_graphblas/algorithms.py

```diff
@@ -1,14 +1,7 @@
 import numpy as np
 from typing import List
-from .functions import (
-    ConvertLayout,
-    MatrixSelect,
-    MatrixReduceToScalar,
-    Apply,
-    MatrixMultiply,
-)
 from mlir_graphblas.mlir_builder import MLIRVar, MLIRFunctionBuilder
-from mlir_graphblas.types import AliasMap, SparseEncodingType, Type
+from mlir_graphblas.types import AliasMap, SparseEncodingType, AffineMap
 from mlir_graphblas.random_utils import ChooseUniformContext, ChooseWeightedContext
 from .sparse_utils import MLIRSparseTensor
 from .engine import MlirJitEngine
```

Comment:
This wasn't part of this PR, but `MlirJitEngine` isn't actually used in this file, so we should consider removing it.